sec1/src/point.rs

```diff
@@ -210,7 +210,7 @@ where
         }
 
         let (x_bytes, y_bytes) = self.bytes[1..].split_at(Size::to_usize());
-        let x = Array::from_slice(x_bytes);
+        let x = x_bytes.try_into().expect("size invariants were violated");
```

Comment:
Wonder if we could add something to `hybrid-array` to help with this pattern, possibly an implementation of [`split_array`](https://github.com/rust-lang/rust/issues/90091). But this is fine for now.

Reply:
I don't know, the API looks acceptable.